File: frontend/src/app/models/LigneWishlistDto.ts
export class LigneWishlistDto{
    id:number
    articleId:number
    tarif:number
    tva:number
     articleIntitule:string 
    articleCode:string
    dateCreate:Date
    tarifttc:number
    articleImage:string=""
    
    constructor(id:number,articleId:number,articleIntitule:string,articleCode:string,
        tarif:number,tva:number,dateCreate:Date,tarifttc:number){
    this.id=id
    this.articleId=articleId
    this.tarif=tarif
    this.dateCreate=dateCreate
    this.tva=tva
    this.articleIntitule=articleIntitule
    this.articleCode=articleCode
    this.tarifttc=tarifttc
        }
}

File: backend/B2BApplication/DTO/LigneWishlistDto.cs
namespace B2BApplication.DTO
{
    public class LigneWishlistDto
    {
        public int id { get; set; }
        public int articleId { get; set; }
        public decimal tarifttc { get; set; }
        public string articleIntitule { get; set; }
        public string articleCode { get; set; }
        public decimal tarif { get; set; }
        public decimal tva { get; set; }
        
        public string articleImage { get; set; }

        public DateTime dateCreate { get; set; } = DateTime.Now;
        public string? userId { get; set; }
    }
}
